crates/types/src/detected_feet.rs

```diff
@@ -25,4 +25,6 @@ pub struct ClusterPoint {
 pub struct CountedCluster {
     pub mean: Point2<Ground>,
     pub samples: usize,
+    pub lefmost_point: Point2<Ground>,
```

Comment:
Found a typo (not sure why this was not added to the review yesterday).

Reply:
Done